packages/nextjs/src/server/utils.ts

```diff
@@ -136,3 +136,10 @@ export const injectSSRStateIntoObject = (obj: any, authObject: AuthObject) => {
     process.env.NODE_ENV !== 'production' ? JSON.parse(JSON.stringify({ ...authObject })) : { ...authObject };
   return { ...obj, __clerk_ssr_state };
 };
+
+export const handleIsSatelliteBooleanOrFn = (opts: WithAuthOptionsExperimental, url: URL): boolean => {
```

Comment:
Should we add a note here that this function is shared between the remix and nextjs packages so we can move them to `clerk/shared` next week? :)

Reply:
@nikosdouvlis I could do this now, but I remember you mentioning not using clerk/shared because it is not tree shakable yet.